contracts/peg/PegStabilityModule.sol

```diff
@@ -451,7 +451,10 @@ contract PegStabilityModule is
     /// @notice Allocates a portion of escrowed PCV to a target PCV deposit
     function _allocate(uint256 amount) internal {
         _transfer(address(surplusTarget), amount);
-        surplusTarget.deposit();
+
+        /* disabled so that we can deposit to non-pcv deposits
+        // surplusTarget.deposit();
+        */
```

Comment:
this is a huge antipattern to remove as a comment like this imo

Reply:
agreed. if you want to remove, best way would be by making this function virtual and overriding

Reply:
Agreed. Are you saying I should remove it entirely? Leaving it restricts us quite a bit as far as destinations for the excess Fei.

Reply:
Great call @ElliotFriedman , making it virtual and overriding.